src/book/03-guides/04-general/01-learning-resources.mdx

```diff
@@ -0,0 +1,392 @@
+
+</details>
+
+<details>
+<summary> Log and Play Back Data </summary>
+
+**Reading:**
+
+- [NUbook: Data Recording and Playback Guide](/guides/main/data-recording-playback)
+- [NUbook: Logging](/system/subsystems/logging)
+
+</details>
+
+<details>
+<summary> Lab Induction </summary>
+
+The lab induction must be completed before getting swipe card access to the lab.
+
+**Reading:**
+
+- [NUbook: Lab Induction](/team/induction)
+
+**Tutorials:**
+
+- NUbots lab inductions are run for new team members after they join.
+
+</details>
+
+## Behaviour
+
+<details>
+<summary>General</summary>
+
+**Reading:**
+
+- [NUbook](/system/subsystems/behaviour)
+
+**Resources:**
+
+- [GitHub: GameController](https://github.com/RoboCup-Humanoid-TC/GameController)
+- [GitHub: Communication Protocol](https://github.com/RoboCup-Humanoid-TC/RobocupProtocol)
+
+</details>
+
+## DevOps and Quality Assurance
+
+<details>
+<summary> The Build System </summary>
+
+**Reading:**
+
+- [NUbook: Build System](https://nubook.nubots.net/system/foundations/build-system)
+- [NUbook: Configuration and Script System](/system/foundations/config-script)
+- [Modern CMake](https://cliutils.gitlab.io/modern-cmake/)
+- [ArchWiki](https://wiki.archlinux.org/)
+- [Book: Docker Deep Dive](https://www.amazon.com.au/Docker-Deep-Dive-Nigel-Poulton-ebook/dp/B01LXWQUFF)
+
+**Tutorials:**
+
+- [Official CMake Tutorial](https://cmake.org/cmake/help/latest/guide/tutorial/index.html)
+
+</details>
+
+## DevTools
+
+<details>
+<summary> Web Development (External) </summary>
+
+Web Development is used in the creation of [NUbook](/system/tools/nubook), which uses JavaScript, Gatsby and Tailwind, and [NUsight](/system/tools/nusight), which uses TypeScript, React and MobX. By working on these systems and going through these resources, you will gain valuable real world skills and experience that is often sought out in industry.
+
+**Videos:**
+
+- [egghead.io: MobX](https://egghead.io/courses/manage-complex-state-in-react-apps-with-mobx)
+
+**Tutorials:**
+
+- [The Modern JavaScript Tutorial](https://javascript.info/)
+- [Official Node.js Tutorial](https://nodejs.dev/learn)
+- [egghead.io: A Beginner's Guide to React](https://egghead.io/courses/the-beginner-s-guide-to-react)
+- [Official Webpack Tutorial](https://webpack.js.org/guides/getting-started/)
+
+**Resources:**
+
+- [MDN Web Docs: JavaScript Resources](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
+- [Official TypeScript Documentation](https://www.typescriptlang.org/docs/handbook/)
+- [Official React Documentation](https://reactjs.org/)
+- [Official MobX Documentation](https://mobx.js.org/README.html)
+- [Official Gatsby Documentation](https://www.gatsbyjs.com/docs/)
+- [Official Tailwind Documentation](https://tailwindcss.com/docs)
+
+</details>
+
+<details>
+<summary> Web Development (Internal) </summary>
```

Comment:
I am not that enthused by these `(Internal)` and `(External)` headings. It seems to me like that could be a strange distinction for someone unfamiliar with these systems

Reply:
I'm also not that enthused. Also considered separating NUbook and NUsight, but NUbook then clashes with Documentation.

Reply:
Changed it to External Resources and NUbots Resources - how does that sound?